base/strings/types.jl

```diff
@@ -4,32 +4,37 @@
 
 ## substrings reference original strings ##
 
+"""
+    SubString(s::AbstractString, i::Integer, j::Integer=endof(s))
+    SubString(s::AbstractString, r::UnitRange{<:Integer})
+
+Like [`getindex`](@ref), but returns a view into the parent AbstractString `s`
+with the given indices instead of making a copy.
```

Comment:
"with the given indices" would be made a bit more specific.

Reply:
I have made it more specific (earlier text was following documentation of `view`)